cryptoki/Cargo.toml

```diff
@@ -16,7 +16,7 @@ libloading = "0.7.0"
 log = "0.4.14"
 derivative = "2.2.0"
 secrecy = "0.7.0"
-psa-crypto = { git = "https://github.com/parallaxsecond/rust-psa-crypto.git", rev = "8605006d34944fa880edd3d4d347f460c5585747", default-features = false, optional = true }
+psa-crypto = { git = "https://github.com/parallaxsecond/rust-psa-crypto.git", rev = "15ba7cd048923ae7193b268df51c004e8ceff48e", default-features = false, optional = true }
```

Comment:
Do you need to use that commit? That might be a problem when you are trying to compile Parsec

Reply:
Oops, that's what I needed locally, but didn't pull down your recent changes in Parsec.